tests/test_losses.py

```diff
@@ -92,6 +93,23 @@ def test_soft_tversky_score(y_true, y_pred, expected, eps, alpha, beta):
     actual = F.soft_tversky_score(y_pred, y_true, eps=eps, alpha=alpha, beta=beta)
     assert float(actual) == pytest.approx(expected, eps)
 
+@pytest.mark.parametrize(
+    ["y_true", "y_pred", "expected", "eps", "alpha", "beta", "gamma"],
+    [
+        [[1, 1, 1, 1], [1, 1, 1, 1], 0.0, 1e-5, 0.5, 0.5, 1.0], 
+        [[0, 1, 1, 0], [0, 1, 1, 0], 0.0, 1e-5, 0.5, 0.5, 1.0], 
+        [[1, 1, 1, 1], [0, 0, 0, 0], 1.0, 1e-5, 0.5, 0.5, 1.0], 
+        [[1, 0, 1, 0], [0, 1, 0, 0], 1.0, 1e-5, 0.5, 0.5, 1.0],  
```

Comment:
Can you please add a test case with gamma != 1?

Reply:
Hi @qubvel, thanks a lot for your comments ! Since @zifuwanggg mentioned that the loss function is already implemented I'll close this pull request